Datacard/makeDatacardGGTT_new.py

```diff
@@ -132,6 +145,11 @@ def grabSystematics(df, args):
         raise Exception()
       df.loc[idx, df_name(syst["name"], year)] = val
 
+  theoryFactoryType = {}
+  for syst in systematics_ggbbres.theory_systematics:
+    if syst['type'] == 'factory':
+      theoryFactoryType[syst['name']] = factoryType(data[mask],syst)
+  
```

Comment:
What is this doing and why is it needed?

Reply:
Those are used nowhere, I remove it.